ambry-commons/src/main/java/com/github/ambry/compression/BaseCompression.java

```diff
@@ -252,9 +269,9 @@ public int compress(byte[] sourceBuffer, int sourceBufferOffset, int sourceDataS
   private void verifySourceData(byte[] sourceData, int sourceDataOffset, int sourceDataSize) {
     Utils.checkNotNullOrEmpty(sourceData, "sourceData cannot be null");
     Utils.checkValueInRange(sourceDataOffset, 0, sourceData.length - 1,
-      "sourceDataOffset is outside of sourceData.");
+        "sourceDataOffset is outside of sourceData.");
```

Comment:
why change any of this if it will be removed ?

Reply:
IntelliJ auto-styling made the change.  It will be removed.